data/json/overmap/specials.json

```diff
@@ -57,13 +57,11 @@
             { "point":[-1,1,0], "overmap": "motel_2_north"},
             { "point":[0,1,0], "overmap": "motel_3_north"}
         ],
-        "connections" : [
-            { "point" : [0,-1,0] }
-        ],
+        "connections": [ { "point": [ 0, -1, 0 ], "terrain": "road", "existing": true } ],
         "locations" : [ "land", "swamp" ],
-        "city_distance" : [10, 120],
-        "city_sizes" : [1, 12],
-        "occurrences" : [0, 5],
+        "city_distance" : [8, 120],
```

Comment:
I'd suggest that this could be reduced all the way to 0, motels could easily be found on city outskirts.